<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>7680</width>
    <height>4320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <strikeout>false</strikeout>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>STEM Laboratory Desktop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.ico</normaloff>assets/icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="realtime_value">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>250</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(102, 243, 49);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="values_title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TITLE OF VALUE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>937</width>
      <height>575</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="chart_title">
       <property name="text">
        <string>TITLE OF CHART</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>518</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="chart_widget" native="true"/>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>908</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="stop_record">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="connect_menu">
    <property name="title">
     <string>Kết nối</string>
    </property>
    <widget class="QMenu" name="com_selection">
     <property name="title">
      <string>Chọn cổng COM</string>
     </property>
    </widget>
    <widget class="QMenu" name="com_baudrate">
     <property name="title">
      <string>Tốc độ giao tiếp</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="com_selection"/>
    <addaction name="com_baudrate"/>
   </widget>
   <widget class="QMenu" name="show_menu">
    <property name="title">
     <string>Hiển thị trên biểu đồ</string>
    </property>
    <addaction name="show_humidity"/>
    <addaction name="show_temperature"/>
    <addaction name="show_pH"/>
    <addaction name="show_oxi"/>
   </widget>
   <widget class="QMenu" name="settings_menu">
    <property name="title">
     <string>Cài đặt</string>
    </property>
    <widget class="QMenu" name="steps_y_menu">
     <property name="title">
      <string>Bước trục y</string>
     </property>
    </widget>
    <addaction name="steps_y_menu"/>
    <addaction name="close_app"/>
    <addaction name="help"/>
   </widget>
   <addaction name="settings_menu"/>
   <addaction name="connect_menu"/>
   <addaction name="show_menu"/>
  </widget>
  <action name="show_humidity">
   <property name="text">
    <string>Độ ẩm</string>
   </property>
  </action>
  <action name="show_temperature">
   <property name="text">
    <string>Nhiệt độ</string>
   </property>
  </action>
  <action name="show_pH">
   <property name="text">
    <string>Độ pH</string>
   </property>
  </action>
  <action name="show_oxi">
   <property name="text">
    <string>Lượng Oxi</string>
   </property>
  </action>
  <action name="close_app">
   <property name="text">
    <string>Đóng ứng dụng</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>Trợ giúp?</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
